PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/InterstitialAdUnit.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  InterstitialAdUnit.swift
+//  PrebidMobileRendering
+//
+//  Copyright © 2021 Prebid. All rights reserved.
+//
+
+import UIKit
+
+public class InterstitialAdUnit: PBMBaseInterstitialAdUnit {
+
+    @objc public override init(configId: String) {
+        super.init(configId: configId, eventHandler: PBMInterstitialEventHandlerStandalone())
+    }
+
+    @objc public override init(configId: String, minSizePercentage: CGSize) {
+        super.init(
+            configId: configId,
+            minSizePercentage: minSizePercentage,
+            eventHandler: PBMInterstitialEventHandlerStandalone())
+    }
+
+    @objc public override init(configId: String, minSizePercentage:CGSize, eventHandler: Any) {
+        super.init(configId: configId, minSizePercentage: minSizePercentage, eventHandler: eventHandler)
```

Comment:
Let's write similar things in similar way. In the above method the parameters of super.init are placed on own lines, in this case all of them on the single line.